.github/workflows/dockerbuild.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: image-builder
+
+on:
+  push:
+    branches: ['*']
+  pull_request:
+    branches: ['*']
```

Comment:
Why both events? does it make sense?

Reply:
Yes, I can remove the push event as whenever the head of pull_request is updated the workflow will run. I added it to test it on my fork otherwise I need to raise a PR to test it.
Will update this